content/en/real_user_monitoring/mobile_and_tv_monitoring/advanced_configuration/ios.md

```diff
@@ -723,6 +771,72 @@ For example, if the current tracking consent is `.pending`:
 - If you change the value to `.granted`, the RUM iOS SDK sends all current and future data to Datadog;
 - If you change the value to `.notGranted`, the RUM iOS SDK wipes all current data and does not collect future data.
 
+## Add user properties
+
+You can use the `addUserExtraInfo` API to append extra user properties to previously set properties.
+
+```swift
+   public static func addUserExtraInfo(
+       _ extraInfo: [AttributeKey: AttributeValue?],
+       in core: DatadogCoreProtocol = CoreRegistry.default
+   ) {
+       let core = core as? DatadogCore
+       core?.addUserExtraInfo(extraInfo)
+   }
+```
+
+## Data management
+
+The iOS SDK first stores events and only uploads events when the [intake specifications][11] conditions are met.
```

Comment:
Where does it store events?

Reply:
We can mention "first locally stores events" IMO